### PR TITLE
Login failure landing

### PR DESCRIPTION
**Pull request by mbrandonw**
When we detect a GitHub login failure we now take the user to a dedicated page where they can fix the problem themselves, or contact us.

### CODE REVIEW

File: Sources/GitHub/Model.swift [hunk 1]
```diff
@@ -2,19 +2,37 @@ import EmailAddress
 import Foundation
 import Tagged
 
-public struct AccessToken: Codable {
+public struct AccessToken: Codable, Equatable, RawRepresentable {
```
Comment (mbrandonw) [inline Sources/GitHub/Model.swift:5]: -[] separate this into a tagged string (`AccessToken`) and a response payload (`AccessTokenResponse`).

File: Sources/PointFree/Auth.swift [hunk 2]
```diff
@@ -39,48 +45,203 @@ func authMiddleware(
...
+      gitHubUserID: newGitHubUser.id,
+      githubAccessToken: accessToken
+    )
+    await fireAndForget {
+      let email = try await gitHub
+        .fetchEmails(accessToken: AccessToken(accessToken: existingAccessToken))
+        .first(where: \.primary)
+        .unwrap()
+        .email
+      let html = String(decoding: Data(), as: UTF8.self)
```
Comment (mbrandonw) [inline Sources/PointFree/Auth.swift:93]: - [ ] Use the actual email view.

File: Sources/PointFree/Auth.swift [hunk 3]
```diff
@@ -39,48 +45,203 @@ func authMiddleware(
...
+            to: [email],
+            subject: "Your GitHub account has been updated",
+            text: html,
+            html: html,
+            domain: mgDomain
+          )
+        )
+      } catch {
+        print(error)
+        print("!!!")
```
Comment (mbrandonw) [inline Sources/PointFree/Auth.swift:107]: - [ ] use issue reporter

File: Sources/PointFree/Session.swift [hunk 4]
```diff
@@ -87,11 +87,14 @@ extension URLRequest {
...
 
+import GitHub
+
 public struct Session: Equatable {
   public var flash: Flash?
+  public var gitHubAccessToken: AccessToken?
   public var user: User?
 
-  public static let empty = Session(flash: nil, user: nil)
+  public static let empty = Session()
```
Comment (stephencelis) [inline Sources/PointFree/Session.swift:97]: Ha is this even used?

**Review by stephencelis — APPROVED** [review]
Looks good! Just that one TODO if you think it's worth knocking out before merging.

**Pull request merged**